components/generic/ContentCard.vue

```diff
@@ -15,6 +15,7 @@
         :aria-label="title"
         :style="!isRelated && cardImageStyle"
         class="card-img"
+        :class="{ 'is-square' : isSquare }"
```

Comment:
The similar items will always be square, that styling can be done based on its context, so I don't think we need this additional class.

Reply:
Yes, you are right, however, this would only work if we were actually using `<img>` tags. Because we are using background images, the image will fill the container, causing the image not to appear square.

Reply:
I don't see what you mean. You can move the after to card-img and get rid of this additional is-square class in the similar items component, right?  
Eg: 

```
  .card-img {
      border-radius: 0;
     &:after {
        content: '';
        display: block;
        padding-bottom: 100%;
      }
    }
    
   // remove this
   // .is-square {
   // }
```

Reply:
Apologies, I thought you mean to remove the class completely. I have applied your changes for you